rust/processor/src/grpc_stream.rs

```diff
@@ -177,11 +177,58 @@ pub async fn get_stream(
         num_of_transactions = ?count,
         "[Parser] Setting up GRPC stream",
     );
-    let request = grpc_request_builder(starting_version, count, auth_token, processor_name);
-    rpc_client
-        .get_transactions(request)
-        .await
-        .expect("[Parser] Failed to get grpc response. Is the server running?")
+
+    // TODO: move this to a config file
```

Comment:
Can add this to config and just give it a default like this https://github.com/aptos-labs/aptos-indexer-processors/blob/main/rust/processor/src/config.rs#L40